README.md

```diff
@@ -146,126 +138,23 @@ Clone the repository
-🔨 Compiling src/openzeppelin/upgrades/library.cairo
-🔨 Compiling src/openzeppelin/upgrades/presets/Proxy.cairo
-🔨 Compiling src/openzeppelin/account/library.cairo
-🔨 Compiling src/openzeppelin/account/presets/EthAccount.cairo
-🔨 Compiling src/openzeppelin/account/presets/Account.cairo
-🔨 Compiling src/openzeppelin/account/presets/AddressRegistry.cairo
-🔨 Compiling src/openzeppelin/account/IAccount.cairo
-✅ Done
+Compiling lib(openzeppelin) openzeppelin v0.7.0 (~/cairo-contracts/Scarb.toml)
+Compiling starknet-contract(openzeppelin) openzeppelin v0.7.0 (~/cairo-contracts/Scarb.toml)
+Finished release target(s) in 16 seconds
 ```
 
 ### Run tests
 
-Run tests using [tox](https://tox.wiki/en/latest/), tox automatically creates an isolated testing environment:
-
-```bash
-tox
-
-====================== test session starts ======================
-platform linux -- Python 3.7.2, pytest-7.1.2, py-1.11.0, pluggy-1.0.0
-rootdir: /home/readme/cairo-contracts, configfile: tox.ini
-plugins: asyncio-0.18.3, xdist-2.5.0, forked-1.4.0, web3-5.29.0, typeguard-2.13.3
-asyncio: mode=auto
-gw0 [185] / gw1 [185]
-......................................................................................
-......................................................................................
-............    [100%]
-```
-
-### Run Tests in Docker
-
-For M1 users or those who are having trouble with library/python versions you can alternatively run the tests within a docker container. Using the following as a Dockerfile placed in the root directory of the project:
-
-```dockerfile
-FROM python:3.7
-
-RUN pip install tox
-RUN mkdir cairo-contracts
-COPY . cairo-contracts
-WORKDIR cairo-contracts
-ENTRYPOINT tox
-```
-
-After its placed there run:
-
 ```bash
-docker build -t cairo-tests .
-docker run cairo-tests
-```
-
-### Parallel Testing
-
-This repo utilizes the [pytest-xdist](https://pytest-xdist.readthedocs.io/en/latest/) plugin which runs tests in parallel. This feature increases testing speed; however, conflicts with a shared state can occur since tests do not run in order. To overcome this, independent cached versions of contracts being tested should be provisioned to each test case. Here's a simple fixture example:
-
-```python
-from utils import get_contract_class, cached_contract
-
-@pytest.fixture
-def foo_factory():
-    # get contract class
-    foo_cls = get_contract_class('Foo')
-
-    # deploy contract
-    starknet = await Starknet.empty()
-    foo = await starknet.deploy(contract_class=foo_cls)
-
-    # copy the state and cache contract
-    state = starknet.state.copy()
-    cached_foo = cached_contract(state, foo_cls, foo)
-
-    return cached_foo
+scarb test
 ```
 
-See [Memoization](https://docs.openzeppelin.com/contracts-cairo/0.6.1/utilities#memoization) in the Utilities documentation for a more thorough example on caching contracts.
-
-> Note that this does not apply for stateless libraries such as SafeMath.
-
 ## Security
 
 > ⚠️ Warning! ⚠️
```

Comment:
In the warning I wouldn't say "Do not use it in production", but something like "Use it in production at your own risk".

Reply:
The same applies to the content of SECURITY.md.

Reply:
I lean towards the current version, we don't want to suggest production usage (even at other's risk).